Code/LauncherUnified/launcher_generator.cmake

```
@@ -124,7 +124,17 @@ foreach(project_name project_path IN ZIP_LISTS LY_PROJECTS_TARGET_NAME LY_PROJEC
     # After ensuring that we correctly support DPI scaling, this should be switched to "PerMonitor"
     set_property(TARGET ${project_name}.GameLauncher APPEND PROPERTY VS_DPI_AWARE "OFF")
     if(LY_DEFAULT_PROJECT_PATH)
-        set_property(TARGET ${project_name}.GameLauncher APPEND PROPERTY VS_DEBUGGER_COMMAND_ARGUMENTS "--project-path=\"${LY_DEFAULT_PROJECT_PATH}\"")
+        if (TARGET ${project_name})
+            get_target_property(project_game_launcher_additional_args ${project_name} GAMELAUNCHER_ADDITIONAL_VS_DEBUGGER_COMMAND_ARGUMENTS)
+            if (project_game_launcher_additional_args)
+                # Avoid pushing param-NOTFOUND into the argument in case this property wasn't found
+                set(additional_game_vs_debugger_args "${project_game_launcher_additional_args}")
+            endif()
+        endif()
+
+        get_target_property(project_game_launcher_additional_args ${project_name} GAMELAUNCHER_ADDITIONAL_VS_DEBUGGER_COMMAND_ARGUMENTS)
+        set_property(TARGET ${project_name}.GameLauncher APPEND PROPERTY VS_DEBUGGER_COMMAND_ARGUMENTS 
+            "--project-path=\"${LY_DEFAULT_PROJECT_PATH}\" ${additional_game_vs_debugger_args}")
```

Instead of trying to adjust the original `set_property(TARGET ${project_name}.GameLauncher ...` call.
Just add the new logic below the original
```suggestion
        set_property(TARGET ${project_name}.GameLauncher APPEND PROPERTY VS_DEBUGGER_COMMAND_ARGUMENTS "--project-path=\"${LY_DEFAULT_PROJECT_PATH}\"")
        if (TARGET ${project_name})
            get_target_property(project_game_launcher_additional_args ${project_name} GAMELAUNCHER_ADDITIONAL_VS_DEBUGGER_COMMAND_ARGUMENTS)
            if (project_game_launcher_additional_args)
                 set_property(TARGET ${project_name}.GameLauncher APPEND PROPERTY VS_DEBUGGER_COMMAND_ARGUMENTS ${project_game_launcher_additional_args}")
            endif()
        endif()
```

---

Unfortunately, that doesn't work. It ends up inserting a ";" in between each argument:

`--project-path="C:/git/o3de-multiplayersample";--console-command-file="server.cfg"`

---

`APPEND_STRING` can be used instead with a space.
```
 set_property(TARGET ${project_name}.GameLauncher APPEND_STRING PROPERTY VS_DEBUGGER_COMMAND_ARGUMENTS " ${project_game_launcher_additional_args}")
```